_posts/newsletter/2021-06-09-this-month-in-solid.md

```diff
@@ -0,0 +1,36 @@
+
+Next, we heard from Ruben Verborgh. Who broke down the need to know about verifiable credentials (VC). His focus was on connecting the dots between how VC is a standard from the W3C and how it affects Solid Pods and gives you consent back. 
+
+Oz Olivo, the VP of Product at Inrupt, was the final Solid roadmap update. He discussed the next steps Inrupt planning for their version 1.2 release, some of the features fell in line with the theme of this month. 
+
+Our first presenter, Eliott Behar, focused on privacy, Solid, and law converge for the better. He explained how Solid not only meets the standards of General Data Protection Regulation (GDPR) but exceeds them. Eliott went over the six lawful bases that make up the GDPR described, but the most questions are the sixth, third-party processors. Solid makes it possible to see and give permissions to share data with these third parties, but how will the community design this?
+
+Tom Haegemans from [Digita](https://www.digita.ai/) presented data sharing and reuse patterns and the tools to use when thinking about its legality. Tom gave the use case of banks and governments and how we reuse data. When using Solid, we see a new way of accessing this data, like continuous access to different parties. [Digita looked at the patterns](digita.ai/resources) created when figuring out reusing data and who’s involved and tracking each user journey. In the end, we saw how there’s a pattern framework we can use when thinking about the legal considerations. 
+
+Last but not least, Greg Golding and Anton Wiklund, from the [Swedish Public Employment Service](https://www.digg.se/om-oss/nyheter/2021/vardefullt-for-individen-att-fa-okad-insyn-och-kontroll-over-sin-data) showed us how consent could work using Solid. Their [Proof of Concept (PoC)](https://gitlab.com/arbetsformedlingen/individdata/rupoc) was about a job seeker giving and reckoning access to their application to a prospective employer. The work happening around compliance, consent, data reuse, privacy, and law shows promise for the web being a place where individuals have true control over their data and information. 
+
+Each month we dive into new and valuable conversations about how Solid is bettering not only the web but your world. As our world and community begin opening up, we see our lives differently. Technology and data will never be the same, and Solid is changing it for the better. To continue this conversation, join us on the [Solid Forum](https://forum.solidproject.org/). Want to participate in these conversations and present at a future Solid World? Fill out the [Solid World speaker form](https://es1cz4pb7oi.typeform.com/to/nietD34f). Lastly, the Solid Team can address any questions, comments, or concerns at info@solidproject.org. 
```

Comment:
I'm not sure I'd say "As our world and community begin opening up" here, with DEI in mind - most of the world is still quite far from opening up, unfortunately.

Also, to make the email address easily copyable:

```suggestion
Each month we dive into new and valuable conversations about how Solid is bettering not only the web but your world. As our world and community begin opening up, we see our lives differently. Technology and data will never be the same, and Solid is changing it for the better. To continue this conversation, join us on the [Solid Forum](https://forum.solidproject.org/). Want to participate in these conversations and present at a future Solid World? Fill out the [Solid World speaker form](https://es1cz4pb7oi.typeform.com/to/nietD34f). Lastly, the Solid Team can address any questions, comments, or concerns at [info@solidproject.org](mailto:info@solidproject.org). 
```